### PR TITLE
Add Dexter-server Swagger documentation

### PR DESCRIPTION
**Pull request by VeronikaSanko**
Create dexter-server specification with Swagger

dexter-server-spec

api_test

### CODE REVIEW

File: project/dexter-server/package.json [hunk 1]
```diff
@@ -8,14 +8,18 @@
...
     "bluebird": "^3.3.0",
+    "body-parser": "1.9.x",
     "express": "3.5.1",
     "json-bigint": "^0.1.4",
     "line-reader": "^0.2.4",
     "lodash": "^4.0.0",
+    "minimist": "*",
     "moment": "^2.5.1",
     "mysql": "^2.11.1",
+    "package.json": "^2.0.1",
```
Comment (KarolAntczak) [inline project/dexter-server/package.json]: What is this package for?

File: project/dexter-server/swagger.json [hunk 2]
```diff
@@ -0,0 +1,172 @@
+{
```
Comment (KarolAntczak) [inline project/dexter-server/swagger.json]: This looks like an example file. Perhaps it should be deleted.

**Comment by KarolAntczak** [comment]
Look great! :)

I have some general comments, though: 
- please rename folder "dist" into "swagger". Current name is a little confusing
- "dist" contains a lot of 3rd party stuff (images,js,html). I would suggest to to keep only what is necessary to build swagger docs by user and provide instruction in README.md.

File: project/dexter-server/swagger.json (hunk 2 above)
Comment (VeronikaSanko) [reply project/dexter-server/swagger.json]: You are right, I've removed it

File: project/dexter-server/package.json (hunk 1 above)
Comment (VeronikaSanko) [reply project/dexter-server/package.json]: Hmm, I'm not really sure, but after I've removed this nothing changed

File: project/dexter-cs/DexterCS/Src/AnalysisEntityFactory.cs [hunk 3]
```diff
@@ -1,4 +1,4 @@
-﻿#region Copyright notice
```
Comment (KarolAntczak) [inline project/dexter-cs/DexterCS/Src/AnalysisEntityFactory.cs]: Changes in this file are unnecessary.

File: project/dexter-cs/DexterCS/Src/CLIAnalysisResultHandler.cs [hunk 4]
```diff
@@ -1,4 +1,4 @@
-﻿#region Copyright notice
```
Comment (KarolAntczak) [inline project/dexter-cs/DexterCS/Src/CLIAnalysisResultHandler.cs]: Changes in this file are unnecessary.

File: project/dexter-cs/DexterDepend/Src/DexterDependPlugin.cs [hunk 5]
```diff
@@ -130,7 +130,6 @@ public void Dispose()
         {
             //TODO:
         }
-
```
Comment (KarolAntczak) [inline project/dexter-cs/DexterDepend/Src/DexterDependPlugin.cs]: Changes in this file are unnecessary.

**Comment by KarolAntczak** [comment]
Please add short notice to README.md, regarding swagger, including how to run it and on what port it is available.

**Pull request merged**